modules/ln_glcore/glcore.scm

```diff
@@ -226,19 +241,24 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 ;; (glCoreClipPush x1 y1 x2 y2)
 (define (glCoreClipPush . coords)
   (let* ((oldlist glcore:cliplist)
-         (newcoords (if (fx= (length coords) 4) (map flo
-           (list (min (car coords) (caddr coords)) (min (cadr coords) (cadddr coords))
-                 (max (car coords) (caddr coords)) (max (cadr coords) (cadddr coords)))) #f))
-         (newlist (if newcoords (append (list newcoords) oldlist)
-            (if (null? oldlist) oldlist (cdr oldlist))))) 
-  (if (not (null? newlist)) (begin
-    (set! glcore:clipx1 (car (car newlist)))
-    (set! glcore:clipy1 (cadr (car newlist)))
-    (set! glcore:clipx2 (caddr (car newlist)))
-    (set! glcore:clipy2 (cadddr (car newlist)))
-  ))  
-  (set! glcore:cliplist newlist)
- ))
+         (newcoords
+          (if (fx= (length coords) 4)
+              (map lambdanative#flo
```

Comment:
`lambdanative#flo` is not defined, which causes a segfault in `glgui-list` - but what concerns me more is that I don't get a warning during compilation for it?